### PR TITLE
Users and estimations backend

### PR DESCRIPTION
**Pull request by zraza-riis**

### CODE REVIEW

File: app/models/room.py [hunk 1]
```diff
@@ -10,4 +10,23 @@ class Room(db.Model):
 
     def __repr__(self):
         return f'<Room {self.name}>'
+    
+class User(db.Model):
```
Comment (jere-mie) [inline app/models/room.py:14]: every user should be associated with a specific room

**Review by jere-mie — CHANGES_REQUESTED** [review]
see comments

File: app/models/room.py (hunk 1 above)
Comment (zraza-riis) [reply app/models/room.py:14]: should be done

**Review by jere-mie — APPROVED** [review]
LGTM 🚀🚀🚀🚀

**Pull request merged**